test/docker-compose.yml

```diff
@@ -102,6 +102,17 @@ services:
       timeout: 10s
       retries: 5
 
+  memcached:
+    image: memcached:alpine
+    user: 11211
```

Comment:
I assume it's a mistake, setting the user to the port number? 🤔

Reply:
The docker image for memcached has a user with 11211. There is a version of the alpine image that errors when it is run as root.

Reply:
Weird choice by whoever made that image. 🤔 

Anyway, ignore this thread. :)